### PR TITLE
Add ability to autocomplete services for brew-services.

### PR DESCRIPTION
**Pull request by RomanKapitonov**
Added the ability to autocomplete services based on the list of services installed via brew.

### CODE REVIEW

File: plugins/brew/_brew [hunk 1]
```diff
@@ -106,4 +111,28 @@ case "$words[1]" in
...
     _wanted outdated_formulae expl 'outdated formulae' compadd -a outdated_formulae ;;
+  services)
+     _arguments -C \
+      '1: :->command' \
+      '2: :->service' && return 0
+
+    local -a commands
+
+    commands=(
+      'clenaup:Get rid of stale services and unused plists'
```
Comment (exah) [inline plugins/brew/_brew]: You have typo here `cleanup => cleanup`.

**Comment by RomanKapitonov** [comment]
@exah fixed a typo

File: plugins/brew/_brew (hunk 1 above)
Comment (EduardoLeggiero) [inline plugins/brew/_brew]: Hi. This new autocomplete for brew services is a good new feature. Do you have time to fix the typo to merge the pull request? Cheers

Comment (RomanKapitonov) [inline plugins/brew/_brew]: @EduardoLeggiero Fixed the typo.

**Comment by RomanKapitonov** [comment]
Fixed conflicts

**Comment by joshka** [comment]
@RomanKapitonov these changes got dropped as part removing the brew plugin from OMZ at https://github.com/robbyrussell/oh-my-zsh/pull/5190. As a result your tab completion for services may break.

An updated _brew completion script will be available in https://github.com/Homebrew/brew/pull/951 shortly, but it doesn't yet have your changes. I'll take a look an re-implement in the next couple of days (unless you beat me to it ;))

I'm guessing these completions probably belong in a script in the [Homebrew/homebrew-services](/Homebrew/homebrew-services) repo, but there would need to be a core change to support installing completions as part of a tap. Something similar to how man pages are handled currently https://github.com/Homebrew/brew/blob/master/Library/Homebrew/tap.rb#L232.

**Pull request merged**